submarine-cloud/hack/deploy-submarine.sh

```diff
@@ -57,7 +67,10 @@ function install_submarine() {
     if $KUBECTL_BIN get configmap --namespace default | grep submarine-config >/dev/null ; then
       $KUBECTL_BIN delete configmap --namespace default submarine-config
     fi
-    $KUBECTL_BIN create configmap --namespace default submarine-config --from-file=${ROOT}/hack/conf/submarine-site.xml --from-file=${ROOT}/hack/conf/log4j.properties
+    $KUBECTL_BIN create configmap --namespace default submarine-config \
+    --from-file=${ROOT}/hack/conf/submarine-site.xml \
+    --from-file=${ROOT}/hack/conf/log4j.properties \
+    --from-file=${ROOT}/hack/conf/k8s/config
```

Comment:
manifests/submarine-cluster/submarine-server.yaml:80 already have 
```
      volumes:
        - name: submarine-configmap
          configMap:
            name: submarine-config
            items:
              - key: submarine-site.xml
                path: submarine-site.xml
              - key: log4j.properties
                path: log4j.properties
```

Reply:
Sorry for that, I forgot to remove it. In cluster mode, we don't need this.